web/satellite/src/views/register/Register.vue

```diff
@@ -48,7 +49,7 @@
             let referralIds = ids ? JSON.parse(atob(ids)) : undefined;
```

Comment:
Is there any reason we use `route params` to pass ids? Also, I am curious what is the difference between `refUserID` and `partnerID`?

Reply:
The route params is just a preference of how marketing want the referral link look like. I will add a comment to explain the difference between `refUserID` and `partnerID`. PartnerID is part of the `User` struct if it exists on the URL, it means that a partner has referred a user. RefUserID is a existing user's ID, it means that the new user is referred by an existing user instead of a partner